### PR TITLE
Minor advisor test improvements

### PR DESCRIPTION
**Pull request by fviernau**
See individual commits.

### CODE REVIEW

File: plugins/advisors/osv/src/funTest/kotlin/OsvFunTest.kt [hunk 1]
```diff
@@ -59,8 +58,8 @@ class OsvFunTest : WordSpec({
             val packageFindings = osv.retrievePackageFindings(packages).mapKeys { it.key.id.toCoordinates() }
 
             packageFindings.keys shouldContainExactlyInAnyOrder packages.map { it.id.toCoordinates() }
-            packageFindings.keys.forAll { pkg ->
-                packageFindings.getValue(pkg).vulnerabilities shouldNot beEmpty()
+            packageFindings.keys.forAll { coordinates ->
+                packageFindings.getValue(coordinates).vulnerabilities shouldNot beEmpty()
```
Comment (sschuberth) [inline plugins/advisors/osv/src/funTest/kotlin/OsvFunTest.kt:62]: Shouldn't this variable renaming still go to the previous commit?

**Review by sschuberth — CHANGES_REQUESTED** [review]
How about a follow-up commit in this PR that turns `identifierToPackage()` into `fun Identifier.toPackage()`?

**Comment by fviernau** [comment]
> How about a follow-up commit in this PR that turns `identifierToPackage()` into `fun Identifier.toPackage()`?

Currently, `identifierToPackage()` takes a `String`. Do you want to change the type to `Identifier` ?

(Maybe I can do this in a follow-up PR)

**Comment by sschuberth** [comment]
> > How about a follow-up commit in this PR that turns `identifierToPackage()` into `fun Identifier.toPackage()`?
> 
> Currently, `identifierToPackage()` takes a `String`. Do you want to change the type to `Identifier` ?

No, you would call it like `Identifier("Maven:org.hamcrest:hamcrest-core:1.3").toPackage()` (instead of `identifierToPackage("Maven:org.hamcrest:hamcrest-core:1.3")`). It's a bit longer, but nicer IMO.

> Maybe I can do this in a follow-up PR

Yes.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**